README.md

```diff
@@ -60,3 +58,16 @@ on why you need to use `127.0.0.1`.
 ```bash
 mysql -h 127.0.0.1 -P 3306 -u deltanyc -ppassword deltanyc
 ```
+
+A note on preg UDFs
+***************************
+The cleanup utils make extensive use of [User Defined preg functions](https://github.com/mysqludf/lib_mysqludf_preg).
+These aren't available in the general MySQL docker container, and the container
+provided by the [uqlibrary](https://github.com/uqlibrary/docker-mysql-udf-preg)
+hasn't been built in a while, so it doesn't have recent fixes to the upstream
+MySQL container.
+
+One challenge here is that the `CREATE FUNCTION` statements need to be called by
+a client, as MySQL UDFs are registered in the `mysql` database. This was solved
+by creating a quick utility container (`db-udf`) that will come up with
+docker-compose, pull the relevant `CREATE` commands, run them, and exit.
```

Comment:
Yep. 

Sadly, on subsequent runs of plain old `docker-compose up`, the `db-udf` container will exit with an error. MySQL will error out upon trying to create the preg functions again. It'll look like this;

```
db-udf_1  | ERROR 1125 (HY000) at line 3: Function 'lib_mysqludf_preg_info' already exists
```

For now, I think it's OK, but I've created an issue for it to be cleaned up.
https://github.com/chrishenry/deltanyc-heatseek/issues/39